src/modules/referral/services/queries.ts

```diff
@@ -47,49 +47,30 @@ export const getTotalReferralRewardsQuery = () => {
 export const getReferreeWalletsQuery = () => {
   return `select count(*) from (
     select distinct on (d."depositorAddr") d."depositorAddr"
-    from deposit d
-    where d."referralAddress" = $1 and
-          d."depositDate" is not null and
-          d."tokenId" is not null and
-          d."priceId" is not null and
-          d.status = 'filled'
+    from deposits_mv as d
+    where d."referralAddress" = $1
```

Comment:
all the queries are basically the same, except that the target table is the `deposits_mv` materialized view